### DISCUSSION TITLE
How to map different folders from two or more drives to individual users?

### DISCUSSION THREAD

**Discussion by yashodhank** [devld/go-drive#51] [General]
is there any way possible to map different folders from s3 bucket or assign individual bucket to each user or group?
For Example:
I have one FTP server with specific path for each user, it works well when I configure root path for each user.
but I also want to give access to AWS S3 bucket in which each user will have their own folders visible and accessible to them only.
Or if I create separate bucket for each user and add it as drive and assign it to user or group?
How do both do that?

Thank you

**Comment by devld** [comment]
This can currently only be done with path mounts and permission control.

You can mount the folders in s3 separately (right click on the folder and select `Mount To`) to the root directory and then set permissions for different users separately.
